content/js3/blocks/promises/index.md

```diff
@@ -0,0 +1,9 @@
++++
+title = 'What is a Promise?'
```

Comment:
Is this block meant to have content? I think content would be useful here, but if not, let's remove it?

Reply:
yes it's for week 4 now -- we reordered